app/controllers/stories_bulk_destroy_controller.rb

```diff
@@ -0,0 +1,24 @@
+class StoriesBulkDestroyController < ApplicationController
+  def create
```

Comment:
why don't use delete verb ?

Reply:
Because delete verb doesn't have body to pass the params

Reply:
Agreed, delete it's just a get request behind the scenes, it would be weird, to have an url `/stories_bulk_destroy?ids=1,2,3,4,5` and then having to split that string param, kind of a hack for something that should have been a post instead. Hence the controller name being a Destroy creation

Reply:
Understood, Sorry I didn't see the params